doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -57,12 +59,12 @@ on the cloud.
 #. Log in to `GitHub`_.
 #. Log in `CircleCI`_ using your GitHub account.
 #. Back in GitHub, at the bottom of the PR, select “Show all Checks”.
-#. Next to “ci/circleci: build_docs”, select “Details”.
-#. Select the “Artifacts” tab and find ``index.html``.
+#. Next to “ci/circleci: build_docs artifact”, select “Details”.
```

Comment:
Well, I thought this is how it's supposed to work with gh-10706, but it's not looking like that for me yet.

Reply:
Update: It will if PR is rebased on master since gh-10706 merged.